lenskit-eval/build.gradle

```diff
@@ -39,6 +39,7 @@ apply from: "$rootDir/gradle/test-utils.gradle"
 
 dependencies {
     compile project(':lenskit-core')
+    compile project(':lenskit-knn')
```

Comment:
The problem is that exception is reproducible only with this particular scorer. So the change to ``testCompile`` dependency would be sufficient!